install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```diff
@@ -48,18 +48,15 @@ spec:
         - |
           rm -rf /var/run/etcd/*;
           /usr/local/bin/etcd --data-dir=/var/run/etcd --name=clustermesh-apiserver --listen-client-urls=http://127.0.0.1:2379 --advertise-client-urls=http://127.0.0.1:2379 --initial-cluster-token=clustermesh-apiserver --initial-cluster-state=new --auto-compaction-retention=1 &
```

Comment:
I've always thought this initContainer was awkward. While you're here, do you want to move this to a postStart hook?

Reply:
I agree that it is awkward, but I don't think that is possible, since using a postStart hook would leave a time window with no RBACs enforcement during startup. It currently does not happen since the init container configures etcd to listen only on localhost, while it is exposed publicly when started by the main container. IMHO, the best solution would be if etcd provided the possibility to RBAC parameters through a config file, but that is not possible as of today.

Reply:
Yeah, makes sense. Annoying.